yarn.lock

```diff
@@ -1514,6 +1514,11 @@ setprototypeof@1.1.1:
   resolved "https://registry.yarnpkg.com/setprototypeof/-/setprototypeof-1.1.1.tgz#7e95acb24aa92f5885e0abef5ba131330d4ae683"
   integrity sha512-JvdAWfbXeIGaZ9cILp38HntZSFSo3mWg6xGcJJsd+d4aRMOqauag1C63dJfDw7OaMYwEbHMOxEZ1lqVRYP2OAw==
 
+shortcut-buttons-flatpickr@^0.3.1:
+  version "0.3.1"
+  resolved "https://registry.yarnpkg.com/shortcut-buttons-flatpickr/-/shortcut-buttons-flatpickr-0.3.1.tgz#25ec705db8e7c80f0f81f5e160783102775ef529"
+  integrity sha512-zZRLJXF7WSeQJ0ZES+PdD1FI+yjBOufqOQ4AU5przR71Rhyekiw/6/JLA33zOqVkgR7QQ1NbVcuuSqNHHGneEg==
+
```

Comment:
ah so, it doesn't directly depend on the flatpickr package. I guess then it's very unlikely we'll run into issues due to having `flatpickr` managed by Bundler and this plugin managed by Yarn.

Reply:
You're right: let's put `flatpickr` in Yarn.

Reply:
Commit 42f6ed0 should resolve this. Thanks Pau!